Automattic/jetpack
# [RNMobile] Android player controls

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5646
Depends on https://github.com/WordPress/gutenberg/pull/49663

## Proposed changes:

The following apply to Android only
* Disables the player control
* Adds basic play, pause, and replay functionality.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






On Android
* Go to a post with an existing VideoPress block
* Notice the play button on the video cover
* Select the block
* Touch the video
* The video should start playing.
* The control icon should change to a pause icon and disappear after short duration (800ms)
* Touch the video before it ends
* The control icon should reappear and change back to the play icon
* Touch the video
* The video should start playing 
* Allow the video to finish (Note: make sure looping is turned off in the block settings)
* Notice the replay icon is displayed
* Touch the video
* The video should start playing from the beginning and the pause button should flash 

### Uploading a new video
* Add an new VideoPress block and choose a video from device
* Notice the play button is not displayed while the video is being processed
* Notice the play button is displayed once the video loads (Note: these changes do not address upload cropping ) 

On iOS
* Verify that the player behaves correctly and is using the built-in player controls
